charts/karmada/templates/karmada-controller-manager.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
   labels:
   {{- include "karmada.cm.labels" . | nindent 4 }}
   annotations:
-    "helm.sh/hook": post-install
+    "helm.sh/hook": post-install,post-upgrade
```

Comment:
Ref to #3813, I think deleting may be better.

Reply:
Ok, I'll delete it